src/arg_spec.ml

```diff
@@ -6,6 +6,7 @@ type 'a t =
   | A        of string
   | As       of string list
   | S        of 'a t list
+  | Tight    of 'a t list
```

Comment:
I debated just calling this `T` 🙂 Another option would be to make it a parameter of `S` (which would have stopped the initial risk of not gathering dependency information in an earlier version) but would be a much more invasive change.

Reply:
What about `Concat of string (* separator *) * 'a t list`?